tests/halcompile/personalities_mod/test.sh

```diff
@@ -1,17 +1,11 @@
 #!/bin/sh
-DIR=../../../src/hal/components ;# use in-tree components
-halcompile --personalities=2 --install $DIR/lincurve.comp
-halcompile --personalities=2 --install $DIR/logic.comp
-halcompile --personalities=2 --install $DIR/bitslice.comp
+${SUDO} halcompile --personalities=2 --install lincurve_test.comp
+${SUDO} halcompile --personalities=2 --install logic_test.comp
+${SUDO} halcompile --personalities=2 --install bitslice_test.comp
```

Comment:
This won't work unless the user has passwordless sudo set up, right?

Can we put in an executable `skip` file `sudo -n true > /dev/null` so that in that situation the test is skipped instead of failing or, worse, pausing for user input?

Reply:
That is correct.  Great idea.

Reply:
Would love to see you add this (the sudo-skips) if you get motivated.  Thanks for the PR!